rosplan_planning_system/src/PlannerInterface/LPGPlannerInterface.cpp

```diff
@@ -93,10 +93,13 @@ namespace KCL_rosplan {
                 // consume useless lines
             }
             planDuration = 0;
-			ss.str("");
+            if ((line.length()>1)){
```

Comment:
This part (lines 96-105) could use some inline comments to explain that it is saving the final block of uninterrupted lines.